modules/bonita-overview/pages/bonita-bpm-overview.adoc

```diff
@@ -62,15 +62,12 @@ xref:#bcd[Bonita Continuous Delivery] comes in very handy to help with this step
 
 Bonita Runtime is composed of one or more Bonita Server nodes installed on some hosts (physical machine, virtual machine, a cloud instance, a container like Docker, ...). A Bonita Runtime composed of multiple nodes constitutes a cluster. +
 
-Bonita provided applications are embedded in the Bonita Runtime. +
+Bonita provided mandatory and optional ressources in the Bonita Runtime:
```

Comment:
"resources" is the right term? I only see applications below
```suggestion
Bonita provides mandatory and optional resources in the Bonita Runtime:
```

Reply:
"Ressources" is the future term that will be used. An application as a ressource that can be mandatory or optional. I may be changing things too early. But at least it will be done.